icubam/backoffice/handlers/plot.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import tornado.web
+import math
+from collections import defaultdict
+
+from bokeh.embed import components
+from bokeh.plotting import figure
+from icubam.backoffice.handlers import base
+
+from pprint import pprint
+
+
+class PlotHandler(base.BaseHandler):
+  ROUTE = '/plot'
+
+  @tornado.web.authenticated
+  async def get(self):
+    """Serves a page with a table gathering current bedcount data with some extra information."""
+    data = {}
+    figures = []
+
+    bed_counts = self.db.get_visible_bed_counts_for_user(None, force=True)
```

Comment:
could you not force and pass as only argument self.user.user_id ? this way the plot will be customized for the user.